app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -576,6 +577,16 @@ public void onReply(int position) {
     @Override
     public void onReblog(final boolean reblog, final int position) {
         final Status status = statuses.get(position).asRight();
+        boolean confirmReblogs = PreferenceManager.getDefaultSharedPreferences(getContext())
+                .getBoolean("confirmReblogs", false);
+        if (confirmReblogs) {
+
+        } else {
```

Comment:
looks like something is mising here

Reply:
I find it funny that you missed that 's' first :laughing:

Reply:
:D